code/game/objects/items/weapons/stock_parts.dm

```diff
@@ -29,7 +29,8 @@
 				M.exchange_parts(user, src)
 				user.Beam(M,icon_state="rped_upgrade", icon='icons/effects/effects.dmi', time=5)
 		else
-			message_admins("\[EXPLOIT] [key_name_admin(user)] attempted to upgrade machinery with a BRPED via a camera console. (Attempted range exploit)")
```

Comment:
This message was put in so we could note down the people previously using this exploit.

I would rather you fix the range thing to add +2 to it to handle people who move quickly as opposed to removing the exploit log.

Reply:
requested changes implemented